tests/app/services/test_response_formatters.py

```diff
@@ -17,18 +19,24 @@
     SEARCH_RESULTS_JSON = json.load(search_results)
 
 
-def test_should_build_query_block_in_response(services_mapping):
+@mock.patch('app.main.services.response_formatters.current_app')
+def test_should_build_query_block_in_response(current_app, services_mapping):
     res = convert_es_results(services_mapping, SEARCH_RESULTS_JSON,
                              {"q": "keywords", "category": "some catergory"})
     assert res["meta"]["query"]["q"] == "keywords"
     assert res["meta"]["query"]["category"] == "some catergory"
 
 
-def test_should_build_search_response_from_es_response(services_mapping):
+@mock.patch('app.main.services.response_formatters.current_app')
+def test_should_build_search_response_from_es_response(current_app, services_mapping):
+    current_app.config.__getitem__.side_effect = mock_config.__getitem__
```

Comment:
Just assigning `mock_config` to `current_app.config` seems to work for me here. What's the reason to prefer assigning `__getitem__` instead?

```suggestion
    current_app.config = mock_config
```

Reply:
Whoops, good spot - that was left over from trying something else! Also previously I thought I was going to need it in multiple tests but that's no longer the case, so I'll assign directly